### PR TITLE
feat(examples) : adding cache control and environmental variable changes

### PR DESCRIPTION
**Pull request by srajak8**
Added cache control changes and environment variables to get the username and password.

### CODE REVIEW

File: examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AfiOptimizerExample.cs [hunk 1]
```diff
@@ -12,9 +12,9 @@ namespace FactSet.AnalyticsAPI.Engines.Example.Examples
     public class AfiOptimizerExample
     {
         private static Configuration _apiConfiguration;
-        private const string BasePath = "https://api.factset.com";
-        private static readonly string UserName = Environment.GetEnvironmentVariable("ANALYTICS_API_USERNAME_SERIAL");
-        private static readonly string Password = Environment.GetEnvironmentVariable("ANALYTICS_API_PASSWORD");
+        //max-stale=0 will be a fresh adhoc run and the max-stale value is in seconds.
+        //Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is 5 minutes older.
+         private static string CacheControl = "max-stale=0";
```
Comment (ksreeramoj) [inline examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AfiOptimizerExample.cs:17]: Can you move it to constants as you are using it in multiple places.

File: examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AxpOptimizerExample.cs [hunk 2]
```diff
@@ -119,9 +119,9 @@ private static Configuration GetApiConfiguration()
 
             _apiConfiguration = new Configuration
             {
-                BasePath = BasePath,
-                Username = UserName,
-                Password = Password
+                BasePath = Environment.GetEnvironmentVariable("FACTSET_HOST"),
+                Username = Environment.GetEnvironmentVariable("FACTSET_USERNAME"),
+                Password = Environment.GetEnvironmentVariable("FACTSET_PASSWORD"),
```
Comment (AkankshaAcharya) [inline examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AxpOptimizerExample.cs:124]: can you remove the extra comma after password?

**Review by ksreeramoj — APPROVED** [review]

File: examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AfiOptimizerExample.cs [hunk 3]
```diff
@@ -24,15 +24,19 @@ public static void Main(string[] args)
                 var afiStrategy = new AFIOptimizerStrategy(null, strategyId);
                 var tradesList = new OptimizerTradesList(OptimizerTradesList.IdentifierTypeEnum.Asset, false);
                 var outputTypes = new OptimizerOutputTypes(tradesList);
+                
+
```
Comment (saigiridhar21) [inline examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AfiOptimizerExample.cs:28]: Can you please remove these extra empty lines?

File: examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AfiOptimizerExample.cs [hunk 4]
```diff
@@ -117,7 +121,7 @@ private static Configuration GetApiConfiguration()
                 BasePath = BasePath,
                 Username = UserName,
                 Password = Password
-            };
+        };
```
Comment (saigiridhar21) [inline examples/FactSet.AnalyticsAPI.Engines.Example/Examples/AfiOptimizerExample.cs:124]: Very minor: Please fix the indentation.

File: examples/FactSet.AnalyticsAPI.Engines.Example/Examples/BpmOptimizerExample.cs [hunk 5]
```diff
@@ -117,7 +119,7 @@ private static Configuration GetApiConfiguration()
                 BasePath = BasePath,
                 Username = UserName,
                 Password = Password
-            };
+        };
```
Comment (saigiridhar21) [inline examples/FactSet.AnalyticsAPI.Engines.Example/Examples/BpmOptimizerExample.cs:122]: Very minor: Please fix the indentation.

**Review by saigiridhar21 — APPROVED** [review]
@srajak8 Left some minor comments

**Review by saigiridhar21 — APPROVED** [review]

**Review by Ramanth — APPROVED** [review]

**Pull request merged**